### PR TITLE
Read and connect timeout are now correctly used to configure the client.

### PR DESCRIPTION
**Pull request by iocanel**

### CODE REVIEW

File: src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesFactoryAdapter.java [hunk 1]
```diff
@@ -40,20 +40,24 @@
     private final StandardCredentials credentials;
 
     private final boolean skipTlsVerify;
+    private final int connectTimeout;
+    private final int readTimeout;
 
     public KubernetesFactoryAdapter(String serviceAddress, @CheckForNull String caCertData,
-                                    @CheckForNull String credentials, boolean skipTlsVerify) {
-        this(serviceAddress, null, caCertData, credentials, skipTlsVerify);
+                                    @CheckForNull String credentials, boolean skipTlsVerify, int readTimeout, int connectTimeout) {
```
Comment (carlossg) [inline src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesFactoryAdapter.java]: Can you keep backwards compatibility?

File: src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesFactoryAdapter.java [hunk 2]
```diff
@@ -40,20 +40,24 @@
...
+        this(serviceAddress, null, caCertData, credentials, skipTlsVerify, connectTimeout, readTimeout);
     }
 
 
     public KubernetesFactoryAdapter(String serviceAddress, String namespace, @CheckForNull String caCertData,
-                                    @CheckForNull String credentials, boolean skipTlsVerify) {
+                                    @CheckForNull String credentials, boolean skipTlsVerify, int connectTimeout, int readTimeout) {
```
Comment (carlossg) [inline src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesFactoryAdapter.java]: Can you keep backwards compatibility?

**Comment by iocanel** [comment]
Tests are failing for reasons not related with the pull requests.

**Comment by carlossg** [comment]
Unfortunately I'm on vacation without my laptop and can't fix it, and github doesn't let me merge from the phone :(

> Pull requests that have a failing status can’t be merged on a phone.

**Comment by iocanel** [comment]
No worries! Have fun!!!

**Pull request merged**